conf/application.conf

```diff
@@ -80,6 +80,13 @@ microservice {
           port = 9480
           startUrl = "transit-movements-trader-at-destination"
         }
+
+      referenceData {
```

Comment:
Do you think we should stick to hyphen convention rather than camelcase for splitting words?